extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/sensors/LightLevelHandler.java

```diff
@@ -61,7 +65,12 @@ protected void doSensorStateChanged(@Nullable HueBridge bridge, @NonNull FullSen
         boolean daylight = Boolean.parseBoolean(String.valueOf(oDaylight));
         updateState(HueBindingConstants.CHANNEL_DAYLIGHT, daylight ? OnOffType.ON : OnOffType.OFF);
 
-        updateState(HueBindingConstants.CHANNEL_LIGHTLEVEL, new DecimalType(String.valueOf(oLightLevel)));
+        // calculate lux, according to
+        // https://developers.meethue.com/documentation/supported-sensors#clip_zll_lightlevel
+        BigDecimal lightlevel = new BigDecimal(String.valueOf(oLightLevel));
+        double lux = Math.pow(10, (lightlevel.subtract(new BigDecimal(1)).divide(new BigDecimal(10000))).doubleValue());
+
+        updateState(HueBindingConstants.CHANNEL_LIGHTLEVEL, new QuantityType<Illuminance>(lux, SmartHomeUnits.LUX));
```

Comment:
You used a QuantityType here. So this comment is outdated https://github.com/eclipse/smarthome/pull/5996#discussion_r211188628